revpi_core.c

```diff
@@ -222,7 +222,8 @@ static void deinit_sniff_gpios(void)
 	piIoComm_writeSniff2A(enGpioValue_Low, enGpioMode_Input);
 
 	if (piDev_g.machine_type == REVPI_CORE ||
-	    piDev_g.machine_type == REVPI_CORE_SE) {
+	    piDev_g.machine_type == REVPI_CORE_SE ||
+	    piDev_g.machine_type == REVPI_CONNECT_4) {
```

Comment:
Change the if-clause to check for !only_left_pibridge in 8c6996e5eedc?

Same for the hunk below.

Reply:
Thanks, missed this one.